elife/pypi.sls

```diff
@@ -15,6 +15,5 @@ pypi-credentials-ubuntu-user:
             - deploy-user
 
 twine:
-    pip.installed:
-        - pkgs:
-            - twine
+    cmd.run:
+        - name: pip install twine
```

Comment:
Did you want to pin a version of this? (ref: https://pypi.org/project/twine/#history )

Reply:
At the moment it doesn't pin a version, by default I'm keeping the same behavior. Perhaps pinning `1.x` would be a good idea.